hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -55,6 +56,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
```

Comment:
If suggestion to use list is accepted, this becomes unused.

```suggestion
```

Reply:
Updated. PTAL.